### PR TITLE
The path is evaluated in "ToLowerInvariant" and the ui shows password and username.

### PR DESCRIPTION
**Pull request by mossbeard**

### CODE REVIEW

File: src/QlikConnectorPSExecute/PSExecuteServer.cs [hunk 1]
```diff
@@ -52,12 +54,18 @@ public override string HandleJsonRequest(string method, string[] userParameters,
...
+                
                 switch (method)
                 {
                     case "getVersion":
                         response = new Info { qMessage = GitVersionInformation.InformationalVersion };
                         break;
+                    case "getUsername":
+                        response = new Info { qMessage = connection.MParameters["UserId"]};
+                        break;
+                    case "getFakePassword":
```
Comment (konne) [inline src/QlikConnectorPSExecute/PSExecuteServer.cs]: no, you can just add the "FakePassword" in the client side content

File: src/QlikConnectorPSExecute/web/connectdialog.ts [hunk 2]
```diff
@@ -74,7 +102,7 @@ class ConnectDialog {
             this.connectionInfo = "Please enter a name for the connection.";
         } else {
             if (this.isEdit) {
-                var overrideCredentials = this.username !== "" && this.password !== "";
+                var overrideCredentials = this.username !== this.olduser || this.password.length !== this.passlength;
```
Comment (konne) [inline src/QlikConnectorPSExecute/web/connectdialog.ts]: this is not working for users with a passwordlength of 4

**Comment by mossbeard** [comment]
@konne I have fixed both points now.

File: src/QlikConnectorPSExecute/web/connectdialog.ts [hunk 3]
```diff
@@ -74,7 +95,7 @@ class ConnectDialog {
             this.connectionInfo = "Please enter a name for the connection.";
         } else {
             if (this.isEdit) {
-                var overrideCredentials = this.username !== "" && this.password !== "";
+                var overrideCredentials = this.username !== this.olduser || this.password.length !== 10;
```
Comment (konne) [inline src/QlikConnectorPSExecute/web/connectdialog.ts]: sorry, we have to discuss it tomorrow again. In this line the algorithm has to check if the new password is != "**********" and not only the length

**Comment by mossbeard** [comment]
@konne Thank you for the note, I have fixed the issue.

**Pull request merged**